libraries/src/Date/Date.php

```diff
@@ -292,7 +292,7 @@ public function calendar($format, $local = false, $translate = true)
 	 *
 	 * @since   1.7.0
 	 */
-	public function format($format, $local = false, $translate = true)
+	public function format($format, $local = false, $translate = true) :string
```

Comment:
Also lets start off right with the correct code style. The colon comes right after the closing bracket, then a space then the type. As per https://wiki.php.net/rfc/return_types

E.g:
```suggestion
	public function format($format, $local = false, $translate = true): string
```